src/documents/adding.md

```diff
@@ -279,17 +194,321 @@ Your flying car has an auto-pilot feature, and you want to add a preference for
             },
-            "autoPilot": {
-                "type": "gpii.firstDiscovery.autoPilot",
+            "autopilot": {
+                "type": "gpii.firstDiscovery.autopilot",
                 "panel": {
-                    "type": "gpii.firstDiscovery.panel.autoPilot",
-                    "container": ".gpiic-fd-prefsEditor-panel-autoPilot",
+                    "type": "gpii.firstDiscovery.panel.autopilot",
+                    "container": ".gpiic-fd-prefsEditor-panel-autopilot",
                     "template": "%templatePrefix/yesNo.html",
-                    "message": "%messagePrefix/autoPilot.json"
+                    "message": "%messagePrefix/autopilot.json"
                 }
             },
             // more content not shown
         }
     });
     ```
+    
+    At this point, you should have a new panel in the tool with a new preference called "Autopilot" that follows the "yesNo" template, and uses the text you specified in the message files. Changing the language setting should use the alternate the text in the panel with the corresponding language selection. 	
+
+    Now that you have a working panel and preference, you'll want to have that preference appear in on the confirm page so that the user can make sure that your new preference is set how they want it.
+     
+6. Edit `src/js.panels.js` to add the selectors for the "Autopilot" preference. 
+
+    A. Find the `"fluid.defaults()"` call for the confirm panel, which calls `"gpii.firstDiscovery.panel.confirm"`.
+    
+    B. In the "selectors" option, you will need to add a preference value and preference label for "Autopilot".
+    
+    ```javascript
+    fluid.defaults("gpii.firstDiscovery.panel.confirm", {
+            gradeNames: ["fluid.prefs.panel"],
+            preferenceMap: {
+                "gpii.firstDiscovery.confirm": {}
+            },
+            selectors: {
+                message: ".gpiic-fd-confirm-message",
+    
+                language: "languageConfirmation",
+                languageLabel: "languageLabel",
+    
+                speak: "speakConfirmation",
+                speakLabel: "speakLabel",
+                
+                autopilot: "autopilotConfirmation",
+                autopilotlabel: "autopilotLabel",
+    
+                speechRate: "speechRateConfirmation",
+                speechRateLabel: "speechRateLabel",
+                
+                //more content not shown
+            },
+            //more content not shown
+    }
+    ```
+    
+7. Edit `src/js.panels.js` to add Friendly Names for the preference. 
+
+    A. In the `"fluid.defaults()"` call to `"gpii.firstDiscovery.panel.confirm"`, find the "friendlyNames" option.
+    
+    B. For each language, you will need to add a new label in the "labels" section for the "Autopilot" preference that follows the same format as the other preferences' friendly names. For unsupported languages, use the English value for the preference. 
```

Comment:
This is OBE by updates the confirmation panel. Friendly names are stored in the JSON message file and transformed with model relays